vaska/js/src/main/scala/vaska/NativeJSAccess.scala

```diff
@@ -19,6 +20,8 @@ final class NativeJSAccess(scope: js.Dynamic) extends JSAccess {
     scala.scalajs.concurrent.JSExecutionContext.runNow
   }
 
+  protected val batchedRequests = mutable.Queue[Request]()
```

Comment:
JSAccess is a bottle neck. I think it better to use js.Array instead mutable.Queue.